### DISCUSSION TITLE
4/5(수) UserDefult

### DISCUSSION THREAD

**Discussion by jjunhaa0211** [Info-iOS/iOStudy#55] [General]
오늘은 쉬어가는...? 타임으로 문법이 아닌 UserDefult swift 실전을 해보도록 하겠습니다. UserDefult는 swift 로컬 데이터 저장 방식 중 하나 입니다.
ToDoList 좀 만들겠다 하면 무조건 사용하는 것이니 알아두기를 바랍니다. 그리고 언제나 어려우면 다온실 또는 나온실로 오시면 저 있습니다. 🥹

**Comment by Eunho0922** [comment]
넵!
